var/spack/repos/builtin/packages/ghostscript/package.py

```diff
@@ -71,3 +73,8 @@ def build(self, spec, prefix):
     def install(self, spec, prefix):
         make('install')
         make('soinstall')
+
+    @classmethod
+    def determine_version(cls, exe):
+        output = Executable(exe)('--version', output=str, error=str)
+        return output
```

Comment:
Can't we look at:
```console
$ gs --help
GPL Ghostscript 9.26 (2018-11-20)
Copyright (C) 2018 Artifex Software, Inc.  All rights reserved.
Usage: gs [switches] [file1.ps file2.ps ...]
[ ... ]
```
and match a regex containing Ghostscript, to be a bit more specific? Asking this since the executable name is just two letters.